web/default/src/styles/index.css

```
@@ -57,11 +44,11 @@ For commercial licensing, please contact support@quantumnous.com
     @apply overflow-x-hidden font-sans;
   }
   body {
-    @apply bg-background text-foreground has-[div[data-variant='inset']]:bg-sidebar min-h-svh w-full;
-    /* Font is driven by the theme's font axis via `--font-body`
-     * (defined in theme.css, swapped by `[data-theme-font='...']` blocks
-     * in theme-presets.css). Defaults to the project's humanist sans. */
-    font-family: var(--font-body);
+    @apply bg-background text-foreground has-[div[data-variant='inset']]:bg-sidebar min-h-svh w-full font-sans;
+    background-image:
+      radial-gradient(circle at 14% 8%, oklch(0.96 0.025 68 / 65%), transparent 38%),
+      radial-gradient(circle at 86% 0%, oklch(0.93 0.04 48 / 35%), transparent 42%);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)